ext/js/language/translator.js

```diff
@@ -219,7 +219,7 @@ export class Translator {
      */
     async _findTermsInternal(text, options, tagAggregator) {
         const {removeNonJapaneseCharacters, enabledDictionaryMap} = options;
-        if (removeNonJapaneseCharacters && options.language === 'ja') {
+        if (removeNonJapaneseCharacters && (['ja', 'zh', 'yue'].includes(options.language))) {
             text = this._getJapaneseOnlyText(text);
```

Comment:
this function internally uses 'isCodePointJapanese', which covers most Japanese punctuation but doesn't have some variants of Chinese punctuation. See 'chinese.js'
We can check their combined code point ranges instead

Reply:
Added